wallet/src/util/ibc-assets.ts

```diff
@@ -0,0 +1,47 @@
+type ChainInfo = {
+  chainName: string;
+  chainId: string;
+  rpc: string;
+  addressPrefix: string;
+  explorerPath: string;
+};
+
+type AssetInfo = {
+  sourcePort: string;
+  sourceChannel: string;
+  denom: string;
+  gas?: string;
+};
+
+type IbcAssets = Record<
+  string,
+  {
+    chainInfo: ChainInfo;
+    deposit: AssetInfo;
+    withdraw: AssetInfo;
+  }
+>;
+
+export const ibcAssets: IbcAssets = {
+  IbcATOM: {
+    chainInfo: {
+      chainName: 'Cosmos Hub',
+      chainId: 'cosmoshub-4',
+      rpc: 'https://cosmoshub-rpc.stakely.io/',
+      addressPrefix: 'cosmos',
+      explorerPath: 'cosmos',
+    },
+    deposit: {
+      sourcePort: 'transfer',
+      sourceChannel: 'channel-405',
+      denom: 'uatom',
+      gas: '100000',
+    },
+    withdraw: {
+      sourcePort: 'transfer',
+      sourceChannel: 'channel-5',
+      denom:
+        'ibc/BA313C4A19DFBF943586C0387E6B11286F9E416B4DD27574E6909CABE0E342FA',
```

Comment:
the denom should be available via `agoricNames.vbankAssets`. I suppose what's deployed doesn't line up, so I can see why you would hard-code this.

Reply:
Yea, maybe we can remove this in favor of that later, but for now this is needed for testing

Reply:
please an `XXX` comment to that effect

Reply:
Done